To-do list/Models/Task.swift

```diff
@@ -7,9 +7,25 @@
 
 import Foundation
 
-struct Task: Codable {
-    var title: String
-    var description: String
-    var deadline: String
-    var status: String
+struct Task: Codable, Equatable {
+    var title: String = ""
+    var description: String = ""
+    var deadline: Date = .now
+    var status: Status = .todo
+    var id: Int = 0
+    
+    enum Status: String, Codable {
+        case todo = "to do"
+        case inProgress = "in progress"
+        case done = "done"
+        
+        var nextState: Task.Status {
+            
```

Comment:
you don't need empty line here. My previous comment about empty line before "switch" control was meant to separate logical blocks of code inside single function. In this case, you have no other logic, but a single switch case. Thus - no empty line needed

Reply:
done